main/src/ca/uwaterloo/flix/language/phase/Weeder2.scala

```diff
@@ -2551,7 +2559,7 @@ object Weeder2 {
       }
     }
 
-    private def tryPickLatticeTermExpr(tree: Tree): Validation[Option[Expr], CompilationMessage] = {
+    private def tryPickLatticeTermExpr(tree: Tree)(implicit flix: Flix): Validation[Option[Expr], CompilationMessage] = {
```

Comment:
implicit flix is no longer used, right?

Reply:
@jaschdoc What do you say?

Reply:
Yes, it's used in the `other` case of `pickRestrictableChoosePattern`